hedera-node/src/main/java/com/hedera/services/store/TypedTokenStore.java

```diff
@@ -245,20 +224,17 @@ private TokenRelationship buildTokenRelationship(
 	private void persistNonDestroyed(
```

Comment:
nit: You can safely remove the last argument from the method as it is a property in the Store